config/storage/key-value/resource-store.json

```
@@ -2,12 +2,23 @@
   "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^1.0.0/components/context.jsonld",
   "@graph": [
     {
-      "comment": "Used for internal storage by some classes. The specified container in the resource store will store internal data.",
-      "@id": "urn:solid-server:default:Storage",
+      "comment": "These storage solutions use the specified container in the ResourceStore to store their data."
+    },
+    {
+      "comment": "Used for internal storage by the locker. Needs to reference the backend store to prevent a loop with the LockingResourceStore.",
```

It took me some thinking work to understand what "backend store" is.

Should we rename into "physical store"?

```suggestion
      "comment": "This is the internal storage for the locker, which maintains what resources are in use. It writes directly to the low-level physical store, because higher-level storage typically already uses the locker and would thus cause a loop.",
```

---

> 
> 
> It took me some thinking work to understand what "backend store" is.
> 
> Should we rename into "physical store"?

It's the store that comes after all our middleware stores. So it can also be a RoutingStore for example if that is used. In that case you would probably not expect that to be called the physical store?

---

If it's a routingstore, yes, it looks like a (virtual) physical store. But I get your point.
Let's leave as-is for now; I will adjust my suggestion.